src/renderer/entities/wallet/model/wallet-model.ts

```diff
@@ -39,6 +39,7 @@ const walletConnectCreated = createEvent<CreateParams<WalletConnectAccount>>();
 
 const walletSelected = createEvent<WalletId>();
 const multisigAccountUpdated = createEvent<MultisigUpdateParams>();
+const walletRemoved = createEvent<Wallet>();
```

Comment:
Why not `WalletId` like we've done in `walletSelected`?
Good to provide as minimum data as needed 😊